datasets/pk/proscribed_persons/crawler.py

```diff
@@ -0,0 +1,33 @@
+from zavod import Context
+from zavod import helpers as h
+
+
+def crawl_person(context: Context, row: dict):
+    person_name = row.pop("name")
+    father_name = row.pop("fatherName")
+    cnic = row.pop("cnic")
+    province = row.pop("province")
+    district = row.pop("district")
+
+    entity = context.make("Person")
+    entity.id = context.make_slug(person_name, prefix="pk-proscribed")
+    entity.add("idNumber", cnic)
```

Comment:
let's make id as make_slug(cnic, prefix="pk-cnic") - we have a similar pattern for "cedula" in some south american countries.

if cnic is currently not set for someone, let's use their name and address components together in make_id.